src/pkg/cli/compose/loader_test.go

```diff
@@ -126,3 +126,14 @@ func TestHasSubstitution(t *testing.T) {
 		})
 	}
 }
+
+func TestNilLoader(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("Nil loader should panic when loading project")
+		}
+	}()
+	var loader *Loader
+	loader.LoadProject(context.Background())
+	t.Error("Should not execute beyond nil loader.LoadProject()")
+}
```

Comment:
This is the kind of test that's written after the fact. Nobody would say "I want this to panic" and then implement the panic.
```suggestion
```